src/patternfly/components/ClipboardCopy/docs/code.md

```diff
@@ -0,0 +1,28 @@
+## Overview
+
+## Accessibility
+
+| Attribute | Applied To | Outcome |
+| -- | -- | -- |
+| `aria-label="[button label text]"` | `.pf-c-clipboard-copy__group-toggle` `.pf-c-clipboard-copy__group-copy` |  Provides an accessible name for the button when an icon is used instead of text. **Required when an icon is used with no supporting text.** |
+| `aria-hidden="true"` | `<i>` |  Hides the icon from assistive technologies. 
+| `aria-controls="[id of expanded element]"` | `.pf-c-clipboard-copy__group-toggle` | Identifies the element controlled by the toggle button. **Required** |
+| `aria-expanded="true"` | `.pf-c-clipboard-copy__group-toggle` | Indicates that the expandable content is visible. |
+| `aria-expanded="false"` | `.pf-c-clipboard-copy__group-toggle` | Indicates that the expandable content is hidden. |
+| `hidden` | `.pf-c-clipboard-copy__expanded-content` | Indicates that the expanded content is hidden so that it isn't visible in the UI and isn't accessed by assistive technologies. |
```

Comment:
```suggestion
| `hidden` | `.pf-c-clipboard-copy__expandable-content` | Indicates that the expandable content is hidden so that it isn't visible in the UI and isn't accessed by assistive technologies. |
```